dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java

```diff
@@ -55,7 +56,7 @@ public void close() {
 
     @Override
     public void configServerPipeline(ChannelPipeline pipeline, SslContext sslContext) {
-        final Configuration config = ConfigurationUtils.getGlobalConfiguration();
+        final Configuration config = ConfigurationUtils.getGlobalConfiguration(ApplicationModel.defaultModel());
```

Comment:
Is the `defaultModel` always ok?

Reply:
I will solve this  in another PR , plz just ignore here

Reply:
LGTM